doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_nordic_bundles.rst

```diff
@@ -0,0 +1,57 @@
+.. _ug_nrf54h20_suit_nordic_bundles:
+
+Upgrade of Nordic Semiconductor firmware components with SUIT
+#############################################################
+
+Apart from the manufacturer application, the devices also contain the Nordic Semiconductor provided components of the firmware, which are used to provide basic functionalities and security features.
+
+These components are provided in the form of the so called "nRF54h20 SOC binaries", which can be found in https://files.nordicsemi.com/ui/repos/tree/General/SDSC
```

Comment:
Can you add a link to the new ABI compatibility page? It indicates the compatibility between NCS versions and soc binaries versions

Reply:
I think this is already done